### ISSUE TITLE
the problem of accuracy

### ISSUE THREAD

**Issue by GCKt** [aioz-ai/MICCAI21_MMQ#11]
Hello, I use the VQA-RAD dataset to train according to the instructions provided by readme, and finally train forty epochs to appear very poor, open only 28 accuracy, close only 70 accuracy, want to ask, is it too little for me to train 40 epoch by default, or may there be problems somewhere, thank you for your help

**Comment by GCKt** [comment]
![Snipaste_2022-10-10_16-43-18](https://user-images.githubusercontent.com/59310845/194838689-2303a36e-46e9-48b9-b515-517d315221cf.png)

**Comment by Gazeal** [comment]
There is some similar problem observed on the PathVQA dataset when other people trained the model on devices that are different than ours.  You can visit [#3](https://github.com/aioz-ai/MICCAI21_MMQ/issues/3) for more information. In this case, the change in initial seeds, learning rate, or max epoch may help. Good luck.

**Comment by GCKt** [comment]
> Hello, I would like to ask you if you have used the vqa-rad dataset to measure the error on different machines, and I will also change the seed learning rate here to try



>

**Comment by GCKt** [comment]
> There is some similar problem observed on the PathVQA dataset when other people trained the model on devices that are different than ours. You can visit [#3](https://github.com/aioz-ai/MICCAI21_MMQ/issues/3) for more information. In this case, the change in initial seeds, learning rate, or max epoch may help. Good luck.

Hello, I would like to ask you if you have used the vqa-rad dataset to measure the error on different machines, and I will also change the seed learning rate here to try，Thanks

**Comment by GCKt** [comment]
> There is some similar problem observed on the PathVQA dataset when other people trained the model on devices that are different than ours. You can visit [#3](https://github.com/aioz-ai/MICCAI21_MMQ/issues/3) for more information. In this case, the change in initial seeds, learning rate, or max epoch may help. Good luck.
I have looked at the accuracy problem mentioned earlier, but I feel that his difference is not very much, I am a little too bad, the difference is close to thirty

**Comment by zhangtianmingxp** [comment]
> 
Hello, I also encountered the same problem. I downloaded the source code and ran the dataset rad and path respectively_ The vqa code is used for training. Under the given configuration, the path_ The precision of vqa dataset is not much worse than that of the paper, and the reproduction effect is very good. But the RAD dataset performs poorly. The open precision is only close to 30. Like the issue author, I want to ask, is it path_ The super parameters trained by vqa dataset and rad dataset are different, and the source code super parameter configuration in GitHub is path_ Vqa, so the effect of running rad with this configuration will be poor. I hope to get a reply. Thank you very much.

**Comment by zhangtianmingxp** [comment]
> There is some similar problem observed on the PathVQA dataset when other people trained the model on devices that are different than ours. You can visit [#3](https://github.com/aioz-ai/MICCAI21_MMQ/issues/3) for more information. In this case, the change in initial seeds, learning rate, or max epoch may help. Good luck.

Hello, do you use this source code to run the precision of the dataset rad? Can the accuracy of the paper be achieved? Thank you very much for your reply.

**Comment by zhangtianmingxp** [comment]
> There is some similar problem observed on the PathVQA dataset when other people trained the model on devices that are different than ours. You can visit [#3](https://github.com/aioz-ai/MICCAI21_MMQ/issues/3) for more information. In this case, the change in initial seeds, learning rate, or max epoch may help. Good luck.

Hello, path_ I have no problems with the vqa dataset. I have seen the third issue. My main problem is vqa_ Rad, do you also use this super parameter for training? Thank you very much for your reply.

**Comment by GCKt** [comment]
Hello, I have visited #3 to see his information, and then I also conducted experiments on the pathvqa dataset and found that the accuracy is basically close to the accuracy of the paper, but I trained the dataset vqa_rad, the accuracy is still much worse, are the hyperparameter configurations of the two datasets the same? Thank you very much for the reply.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "aioz-ai/MICCAI21_MMQ"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月12日(星期三) 晚上10:18
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [aioz-ai/MICCAI21_MMQ] the problem of accuracy (Issue #11)





  
There is some similar problem observed on the PathVQA dataset when other people trained the model on devices that are different than ours. You can visit #3 for more information. In this case, the change in initial seeds, learning rate, or max epoch may help. Good luck.
  
Hello, path_ I have no problems with the vqa dataset. I have seen the third issue. My main problem is vqa_ Rad, do you also use this super parameter for training? Thank you very much for your reply.

**Comment by Gazeal** [comment]
Well, training hyper-params for pathVQA and VQA_RAD are different. Although we have mentioned in the repo [here](https://github.com/aioz-ai/MICCAI21_MMQ#training), we have discovered that we have made some mistakes in the file name, leading to the result that you can not run the training command properly. Sorry for the inconvenience and thank both of you for pointing this out.

**Comment by GCKt** [comment]
Thank you very much for your reply. I have successfully run the vqa rad dataset, but the accuracy is much lower. Could you please provide me with the hyperparameter setting of the training dataset vqa rad?




---Original---
From: "Tuong ***@***.***&gt;
Date: Wed, Oct 12, 2022 23:41 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aioz-ai/MICCAI21_MMQ] the problem of accuracy (Issue #11)




 
Well, training hyper-params for pathVQA and VQA_RAD are different. Although we have mentioned in the repo here, we have discovered that we have made some mistakes in the file name, leading to the result that you can not run the training command properly. Sorry for the inconvenience and thank both of you for pointing this out.

**Comment by GCKt** [comment]
Thank you very much for your reply. I successfully run vqa_ The precision of rad dataset is very poor. Recently, I want to reproduce this paper in vqa_ For the accuracy of the rad dataset, can you provide the super parameter settings for the next training? In addition, you said that there are some mistakes that can be told to me and I will correct them. This will be of great help to me. Thank you very much.



---Original---
From: "Tuong ***@***.***&gt;
Date: Wed, Oct 12, 2022 23:41 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aioz-ai/MICCAI21_MMQ] the problem of accuracy (Issue #11)




 
Well, training hyper-params for pathVQA and VQA_RAD are different. Although we have mentioned in the repo here, we have discovered that we have made some mistakes in the file name, leading to the result that you can not run the training command properly. Sorry for the inconvenience and thank both of you for pointing this out.

**Comment by Gazeal** [comment]
Please reclone the repo, and follow the preprocessing steps again but for the VQA_RAD dataset. And then run `sh run_vqa_VQA_RAD.sh` as mentioned in [here](https://github.com/aioz-ai/MICCAI21_MMQ#training) for training. The training setup for VQA_RAD is in [here](https://github.com/aioz-ai/MICCAI21_MMQ/blob/main/run_vqa_VQA_RAD.sh).

**Comment by GCKt** [comment]
Well, thank you very much. The training results are close to the paper accuracy will tell you.



---Original---
From: "Tuong ***@***.***&gt;
Date: Thu, Oct 13, 2022 00:19 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aioz-ai/MICCAI21_MMQ] the problem of accuracy (Issue #11)




 
Please reclone the repo, and follow the preprocessing steps again but for the VQA_RAD dataset. And then run sh run_vqa_VQA_RAD.sh as mentioned in here for training. The training setup for VQA_RAD is in here.
